### DISCUSSION TITLE
박세빈님의 작업 내용이 변경되었습니다.

### DISCUSSION THREAD

**Discussion by KimMin-Gwan** [KimMin-Gwan/ICT-convergence-project-contest-2023#31] [General]
박세빈님은 오늘 이야기한 대로 제스처 모드에서 명령을 내리는 부분을 작성하시면됩니다.
이전 좌표들을 인과적으로 분석하여 행동을 특정하고, 특정 명령으로 받게 설계하시면됩니다.
자세한 내용이나 필요한 내용, 메모 등을 이곳에 작성하시면 됩니다.

**Comment by KimMin-Gwan** [comment]
박세빈님의 작업이 변경되면서 이현성님의 작업도 변경이 예정되어 있습니다.
특별한 지시가 나올 때 까지 대기하시면 됩니다.

**Comment by Beantree09** [comment]
확인했습니다!

**Reply by KimMin-Gwan** [reply]
새로 작성한 main 폴더의 내용을 확인부탁드립니다.
main/Server/posenet/gesture_mode.py 에 들어갈 내용을 작성해야 합니다.
현재 main은 정상 동작하지 않기 때문에 지금 작업하시던 테스트 폴더에서 비슷한 작업을 수행하시면 됩니다.
